test/core/end2end/tests/retry_lb_fail.cc

```diff
@@ -0,0 +1,282 @@
+//
+// Copyright 2017 gRPC authors.
```

Comment:
s/2017/2021
(looks like a new file to me)

Reply:
I copied this from the existing retry_lb_drop.cc file and just changed some of the details, so I retained the original date.